beets/util/__init__.py

```diff
@@ -829,6 +832,8 @@ def command_output(cmd, shell=False):
             cmd=' '.join(cmd),
             output=stdout + stderr,
         )
+    if include_stderr:
+        stdout += stderr
     return stdout
```

Comment:
What about just returning `stdout, stderr` (when requested, or always) and letting the client pull out what they need?

Reply:
Done by now, forgot to 'resolve conservation'.
We return a `namedtuple` of `stdout` and `stderr`, allowing everyone to choose what they need while being explicit.